Framework/basic.json

```diff
@@ -35,6 +35,22 @@
           "nothing": "rien"
         },
         "location": "remote"
+      },
```

Comment:
Sure, I removed the configuration from `basic.json` and moved it to `Modules/TOF/tof.json`.
However I'm not sure how to include this configuration file in the `runbasic` workflow.

Reply:
Oh yes, as for now, you would need to create your own workflow in the module. You can f.e. copy the `runBasic.cxx` into `Modules/TOF/src/`, change names and refer to your config file instead of the basic.json. To configure CMake to compile your executable, you can have a look how it is done in [the CMakeLists.txt in Framework](https://github.com/AliceO2Group/QualityControl/blob/2ad2ae3663ab1aa24d3ff15c5548a32288c0fcb6/Framework/CMakeLists.txt#L129) and do it similarly in CMakeLists.txt of your module. If you need more detailed instructions, don't hesitate to ask.

Reply:
Ok, I see now, thank you.
I managed to make everything work, however I miss some configuration in the `Modules/TOF/CMakeLists.txt` as I am not able to setup  the correct include path for `runnerUtils.h` and `ExamplePrinterSpec.h`.
I am forced to get them from their relative path as shown in last commit (dc93a07) and I'm afraid this is not ideal ;)

Reply:
Hmm, I can move these two into `include/QualityControl`, so you would be able to refer to them normally. I will make a PR.

Reply:
Nice, thanks!

Reply:
Ok, I have now updated the include path accordingly with commit 8b26ed9.
Can we proceed to close the PR or do you have any other suggestion? 
By the way, thanks a lot for your useful help!!

Reply:
Sorry for the delay, I have been on holidays. Could you rebase it on the top of the master branch and resolve the conflicts? I don't have any more comments to the PR itself.

Reply:
OK, perfect, I mark the change requests as solved then

Reply:
There are still TOFTasks in the basic.json, could you please remove them?